README.md

```
@@ -30,38 +22,36 @@
   <a href="https://discord.gg/6dffbvGU3D" target="_blank">
     <img src="https://img.shields.io/discord/1122926057641742418?logo=discord&logoColor=%23FFFFFF&label=Discord&color=%236963ff" alt="Discord">
   </a>
+  </div>
+  <br>
+  <div align="center" style="display: flex; justify-content: center; gap: 16px; height: 43px; margin: 0;">
+    <a href="https://discord.gg/6dffbvGU3D?ref=github_readme" target="_blank">
+      <img src='./assets/buttons/discord.svg'/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)